scripts/about.js

```diff
@@ -0,0 +1,76 @@
+function about(x) {
```

Comment:
This works well, but remember you can also pass more than one parameter and avoid the hardcoded if statements.